lib/libsys/__getosreldate.c

```diff
@@ -44,23 +44,10 @@ int
 __getosreldate(void)
 {
 	static int osreldate;
```

Comment:
Can this be made non-`static` just like in 9c820a3cf259cc857931c999f3f453b4a2045a5c?

Reply:
`__getosreldate` is used in realpath() which makes a performance enhancing syscall based on it so I decided to leave the optimization in place.